packages/graph/hash_graph/lib/graph/src/store/postgres/query/entity_type.rs

```diff
@@ -31,30 +27,59 @@ impl<'q> PostgresQueryRecord<'q> for EntityType {
 }
 
 impl Path for EntityTypeQueryPath {
-    fn tables(&self) -> Vec<(TableName, EdgeJoinDirection)> {
+    /// Returns the relations that are required to access the path.
+    ///
+    /// The first element of the tuple is the column access of the source table, the second element
+    /// is the name of the target table, and the third element is the column access of the target.
```

Comment:
I think this comment is old, as there are no tuples anymore

Reply:
Good point, removed it